### ISSUE TITLE
Need Aria label for search button

### ISSUE THREAD

**Issue by JSweet314** [posi7790/check-yo-self-practice#15]
Search button in header should have an aria-label since there is only an image and no text. Cannot rely on the image alt for screen readers when in a button.

Side note - if the search is used to 'find' items on the page, then it may be more appropriate to wrap the search in a `<nav>` element, rather than a form.
